File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>


int GetPositiveInteger(void);

int main(void){
 int spaces = 0;//number of spaces
 int hashes = 0;//number of hashhes

 printf("Input:\n");
 int c = GetPositiveInteger();
 for (int i = 1; i<=c; i++) {
  hashes = i + 1;
  spaces = c-hashes+1;
  printf("%.*s", spaces, "                                          ");
  printf("%.*s", hashes, "############################################");//hashes = precision, ################### is argument that has to be formatted
  printf("\n");
}
}
int GetPositiveInteger(void){
  int x;
  do{
   printf("Height?\n");
   x = GetInt();
  }
  while (x < 0 || x >= 24); 
  return x; 

}

File: pset6/mario.py
import cs50

while True:
    print("give height between 0 and 23")
    num = cs50.get_int()
    if num < 23 or num > 0:
        break
    
spaces = num - 1
hashes = 2
i = 0
for i in range(num):
    for j in range(num - i - 1):
        print(" ", end="")
    for k in range(i + 2):
        print("#", end="")
    print()
    
    
    
    
    